scripts/lint-python

```diff
@@ -28,7 +28,7 @@ fi
 # behavior).
 # We must typecheck the whole `py` directory even if only a few files are modified so that mypy
 # sees all types
-${MYPY} --implicit-reexport --namespace-packages --ignore-missing-imports --strict py
+${MYPY} --implicit-reexport --namespace-packages --ignore-missing-imports --strict py py/bitbox02
```

Comment:
why is this needed? doesn't it do `py` and all imported dependencies?

Reply:
It's needed because py/bitbox02 is a package now. So the imports in for example `send_message.py` requires you to actually install bitbox02 before you can use it.